test/scalability/checker/checker_test.go

```diff
@@ -35,10 +35,8 @@ var (
 
 type RangeSpec struct {
 	Cmd struct {
-		MaxWallMs int64  `json:"maxWallMs"`
```

Comment:
Why changing this?

Reply:
To not be tight to absolute values.

Reply:
As in the other comment, I prefer to use the more intuitive values. Can we please just focus this PR on fixing the flake, and bump the values?